engine/runtime/src/main/scala/org/enso/compiler/SerializationManager.scala

```diff
@@ -370,8 +370,8 @@ final class SerializationManager(
           module.setLoadedFromCache(true)
           logger.log(
             debugLogLevel,
-            "Restored IR from cache for module [{0}] at stage [{1}].",
-            Array(module.getName, loadedCache.compilationStage())
+            () =>
```

Comment:
TCR: I find the previous version better. Single text with `{0}` placeholders and an array with values to fill in. Such a style allows for example [structured logging](https://bits.netbeans.org/dev/javadoc/org-openide-util/org/openide/util/doc-files/logging.html). What's the justification for this change?

Reply:
Because the previous result of the logging of Array is, e.g., like this:
```log
[debug] [2023-04-18T10:06:25.573Z] [enso.org.enso.compiler.SerializationManager] Restored IR from cache for module [[Ljava.lang.Object;@2ed09493] at stage [{1}]
```
Probably the `logger.log` is not able to cope with Scala's `Array`? Anyway, I always use lambda functions for loggers. I find it most straightforward and if I understand that correctly, it should also have a lower overhead if the logger is disabled, because you don't have to create any objects.

Reply:
Fixed in 45d9301e899f896752b28db65ce84867453afd4a without using lambda.